caffe2/image/transform_gpu.cu

```diff
@@ -62,4 +61,19 @@ bool TransformOnGPU<uint8_t, float, CUDAContext>(Tensor<CUDAContext>& X, Tensor<
   return true;
 }
 
+template <>
+bool TransformOnGPU<uint8_t, float16, CUDAContext>(Tensor<CUDAContext>& X, Tensor<CUDAContext> *Y, float std, float mean, CUDAContext *context)
```

Comment:
The code is the same as above. What about moving the implementation to the templated version (line:45) ?